shashi/jan27.js

```diff
@@ -0,0 +1,29 @@
+//Assumptions:
+//All names in the arrya are alohabets
+//No Numeric values
+//No special characters
+//No Empty names
+//Any case names
+
+function secretSocityName(arr) {
+  var teamName = "";
+  arr.sort().forEach(name => {
```

Comment:
let's not use sort or forEach or any helper methods.

Reply:
uses Array.prototype.reduce in this case. if you;re going to use an array method use the better option.